packages/compose/src/hooks/use-disabled/index.js

```diff
@@ -53,61 +55,135 @@ const DISABLED_ELIGIBLE_NODE_NAMES = [
  * };
  * ```
  */
-export default function useDisabled() {
-	return useRefEffect( ( node ) => {
-		const disable = () => {
-			node.style.setProperty( 'user-select', 'none' );
-			node.style.setProperty( '-webkit-user-select', 'none' );
-			focus.focusable.find( node ).forEach( ( focusable ) => {
-				if (
-					includes( DISABLED_ELIGIBLE_NODE_NAMES, focusable.nodeName )
-				) {
-					focusable.setAttribute( 'disabled', '' );
-				}
+export default function useDisabled( {
+	isDisabled: isDisabledProp = false,
```

Comment:
This flag disables useDisabled. it's weird I know, I don't know if there's a better way to name this.